package.json

```diff
@@ -3,14 +3,15 @@
   "version": "1.0.0",
   "scripts": {
     "dev": "next",
-    "build": "next build",
+    "build": "NODE_OPTIONS=--openssl-legacy-provider next build",
```

Comment:
what is this doing?

Reply:
theres a dependency that wont build with node 18+ originally this was on 16.  so this allows it to build